component.yaml

```
@@ -8,11 +8,10 @@ inputs:
   isRequired: false
   title: Project Path
   description: The path to the directory that contains the definition of the project. Defaults to current directory.
```

1. I did not understand what do you mean by "definition of the project"
2. IMO this use is more understandable:
  ```suggestion
    description: The path to the directory that contains the definition of the project. By default, it is set to the root directory of the cloned repository.
```